### ISSUE TITLE
Error installing kaltura-nginx on CentOS6

### ISSUE THREAD

**Issue by gfw-breaker** [kaltura/nginx-vod-module#955]
Error message as below:
```
[root@proxy ~]# yum install kaltura-nginx
Loaded plugins: fastestmirror
Setting up Install Process
Loading mirror speeds from cached hostfile
 * base: ftp.tsukuba.wide.ad.jp
 * epel: ftp.riken.jp
 * extras: ftp.tsukuba.wide.ad.jp
 * nux-dextop: mirror.li.nux.ro
 * updates: ftp.tsukuba.wide.ad.jp
http://installrepo.kaltura.org/releases/latest/6/RPMS/noarch/repodata/d15d641babf2c6b266a5d590770ad5e4378c7f6f14c0dafe58dfffb4ec5ef0b8-primary.sqlite.bz2: [Errno 14] PYCURL ERROR 22 - "The requested URL returned error: 404 Not Found"
Trying other mirror.
To address this issue please refer to the below wiki article 

https://wiki.centos.org/yum-errors

If above article doesn't help to resolve this issue please use https://bugs.centos.org/.

Error: failure: repodata/d15d641babf2c6b266a5d590770ad5e4378c7f6f14c0dafe58dfffb4ec5ef0b8-primary.sqlite.bz2 from Kaltura-noarch: [Errno 256] No more mirrors to try.```

**Comment by jessp01** [comment]
Hi @gfw-breaker,

Since installrepo.kaltura.org is a CDN endpoint and we've just released a new Kaltura Server version [14.13.0], it may take a little while for all edges to sync with the origin server.
If you're having issues with fetching http://installrepo.kaltura.org/releases/latest/6/RPMS/noarch/repodata, please edit /etc/yum.repos.d/kaltura.repo and replace 'installrepo.kaltura.org' with 'installrepo.origin.kaltura.org'. 
Following that, try to invoke `yum install kaltura-nginx` again.

Do let me know should you encounter additional issues.

Thanks,

**Comment by jessp01** [comment]
Closing due to lack of response. Feel free to reopen if the issue persists.
